cumulusci.rb

```diff
@@ -44,6 +44,11 @@ class Cumulusci < Formula
     sha256 "05a6052c6a9f17ff78ba78f8e6eb1d777d25db3b763343a1ae89a7a8670386dd"
   end
 
+  resource "cumulusci" do
+    url "https://files.pythonhosted.org/packages/cc/1f/3eb3f8eb9167b82d50ee62483f61cac5bf39617de872edc8af2f48bc6ea0/cumulusci-2.2.5.tar.gz"
+    sha256 "5abdc88e33d6adf0722213e3ddb6e2fb82fbbb8eb0c036e68d0ec5871ff42add"
+  end
```

Comment:
Is cumulusci itself supposed to be here _and_ up above?

Reply:
Yeah, I saw that. `homebrew-pypi-poet` includes that resource stanza, but it didn't affect the installation during my testing, so I left it as is.